components/footer/footer-column.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+interface Props {}
+
+const defaultProps = {}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterColumnProps = Props & typeof defaultProps & NativeAttrs
+
+const FooterColumn: React.FC<React.PropsWithChildren<FooterColumnProps>> = ({
+  children,
+  ...props
+}) => {
+  return <div {...props}>{children}</div>
```

Comment:
We can replace it with `Footer.Group`, there is no need to add another component, if the user adds a nested structure, we just parse the structure.